### DISCUSSION TITLE
how can I setup a admin account to receive all mails

### DISCUSSION THREAD

**Discussion by tanpengsccd** [emersion/maddy#499] [Q&A]
for example 
I create a admin@mydomain.com , then other XXX@mydomain.com's mail can make a copy for admin@mydomain.com when them 
receive new mail.

**Comment by foxcpp** [comment]
Define a second delivery target for local domains, that rewrites any recipient address to admin@mydomain.com.

For example:
```
deliver_to msgpipeline {
  destination admin@mydomain.com {
    deliver_to dummy # do not duplicate messages sent to admin@mydomain.com initially
  }
  default_destination {
    modify { replace_rcpt regexp ".*" "admin@mydomain.com" }
    deliver_to &local_mailboxes
  }
}
```
This would go here, before/after deliver_to: https://github.com/foxcpp/maddy/blob/c9c856ffb6fbe054165acc5e4748df38cd900211/maddy.conf#L73-L79
